src/scripts/chartist-plugin-pointlabels.js

```diff
@@ -32,6 +32,16 @@
             }, options.labelClass).text(options.labelInterpolationFnc(data.value.x === undefined ? data.value.y : data.value.x + ', ' + data.value.y));
           }
         });
+      } else if (chart instanceof Chartist.Bar) {
+        chart.on('draw', function (data) {
+          if (data.type === 'bar') {
+            data.group.elem('text', {
```

Comment:
This line causes problem with stacked bar chart, since it adds labels to layer of data columns/bars. But instead data labels should be in ct-layers i suppose, as suggested in [https://github.com/gionkunz/chartist-js/issues/274#issuecomment-96386894](url).

So changing this line to
`data.element.root().elem('text', {`
Works fine with stacked bars and probably in other coincidences where labels could not seen under graph elements.
![selection_023](https://cloud.githubusercontent.com/assets/9109513/17559544/459e8adc-5f27-11e6-9f61-1b0ae6a62d5d.png)
![selection_025](https://cloud.githubusercontent.com/assets/9109513/17559543/45905660-5f27-11e6-8af1-48d3c0076976.png)

Example screenshots attached: